texts/epn_paper.md

```diff
@@ -0,0 +1,74 @@
+# Let's talk online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine has changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak.
+You now mostly stay at home, hopefully safe and healthy.
+Your classes are now taught online, and you probably had to quickly figure out how to run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or even gave one yourself.
```

Comment:
```suggestion
Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or even given one yourself.
```

Reply:
I think you're missing a 'have' here -> likely attended a scientific presentation or have even given one yourself